### DISCUSSION TITLE
Convert a pre-trained network to analog network.

### DISCUSSION THREAD

**Discussion by matifali** [IBM/aihwkit#429] [General] [answered]
How can we convert a pre-trained network to an analog equivalent Memristive network.?

**Comment by maljoras** [comment, marked as answer]
Hi @matifali, 
thanks for the question. Please take a look at [example 17](https://github.com/IBM/aihwkit/blob/master/examples/17_resnet34_imagenet_conversion_to_analog.py#L45) how to do that. There are conversion functions described [here](https://aihwkit.readthedocs.io/en/latest/api/aihwkit.nn.conversion.html?highlight=convert_to_analog#aihwkit.nn.conversion.convert_to_analog)
